packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -10,3 +10,11 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-one-time-charges = One time charges (due today)
+sub-update-prorated-total = <strong>Upgrade</strong> (Prorated to end of period)
```

Comment:
What's "Upgrade" here? Is it a noun, as in "Cost for the upgrade", or a verb?

Reply:
I thought of it more as an adjective to describe the type of purchase. But there is a price to be added beside it, so then I agree it would be a noun. Do I add a comment above it to make that clear?

Reply:
Yes, please. Something like `Upgrade is a noun, as in "cost for upgrade", and is followed by the price`.